changelog.txt

```diff
@@ -2,6 +2,7 @@
 
 = 7.4.0 - 2024-xx-xx =
 * Update - Schedule subscription-related events with a priority of 1 to allow for earlier execution within the Action Scheduler.
+* Fix - Ensure admin notices are displayed after performing bulk actions on subscriptions when HPOS is enabled.
```

Comment:
@Mayisha Just noting I added this changelog entry as one was missing. Feel free to tweak it. :)

Reply:
Thanks James 🍫